build.sbt

```diff
@@ -1,24 +1,27 @@
 name := "scala-iso"
-scalaVersion := "2.13.3"
-crossScalaVersions := Seq("2.12.13", "2.13.3")
```

Comment:
note - I still would like to cross release (to 2.13 mostly, don't mind about 2.12 tbh) - I think it should work having both  3 and 2_13 in `crossScalaVersions`

Reply:
:+1: